activestorage/app/controllers/concerns/active_storage/streaming.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module ActiveStorage::Streaming
+  private
+
+  # Stream the blob from storage directly to the response. The disposition can be controlled by setting +disposition+.
+  # The content type and filename is set directly from the +blob+.
+  def stream_from_storage(blob, disposition: "inline")
+    response.headers["Content-Type"] = blob.content_type_for_serving
+    response.headers["Content-Disposition"] = ActionDispatch::Http::ContentDisposition.format \
+      disposition: blob.forced_disposition_for_serving || disposition, filename: blob.filename.sanitized
```

Comment:
By omitting the params lookup from the previous `blob.forced_disposition_for_serving || params[:disposition] || disposition`, we're subtlety changing behavior. Was it intended that the controllers would pass it manually? Otherwise I don't quite understand the need for the `disposition: "inline"` override in the method signature.